STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/dm_1_model.h

```diff
@@ -29,14 +29,26 @@ extern "C"
 #define DM_1_INTERFACE_VERSION 1
 #define DM_1_CAPABILITY_SIZE 2
 
+typedef enum 
+{ 
+  DM_1_SOURCE_TYPE_IN_MEMORY = 0, 
```

Comment:
I'm assuming memory here means flash? Might be some confusion between this and the built_in option

Reply:
Yes, for the current MCU, memory means Flash, however, it may not be true for other MCUs, so, we should use Memory because it is more generic. 

Regarding Built_in, we will probably remove it from the code in the future.